src/styles.js

```diff
@@ -5,6 +5,14 @@ const GlobalStyle = createGlobalStyle`
 		margin: 0;
 		padding: 0;
 	}
+	html {
+		box-sizing: border-box;
+	}
+	*,
+	*:before,
+	*:after {
+ 		box-sizing:inherit;
```

Comment:
What's this doing?

Reply:
Popular `box-sizing` reset. Do you know a better one?

Reply:
No but usually I don’t reset at all and do it when needed.

Reply:
I don't want to think about this every time and resetting it globally worked fine for me in the past decade ;-)

Reply:
Yeah I’m fine with it. Was just curious since I never saw it like this.